### PR TITLE
Add third gender

### PR DESCRIPTION
**Pull request by wushin**
Needs M+ modification

https://gist.github.com/wushin/80487aa29080814b681f

### CODE REVIEW

**Comment by wushin** [comment]
@mekolat can you review this.

**Comment by mekolat** [comment]
will do

File: src/admin/ladmin.cpp [hunk 1]
```diff
@@ -1750,6 +1750,8 @@ void parse_fromlogin(Session *s)
                                 PRINTF("%-5s "_fmt, "Femal"_s);
```
Comment (mekolat) [inline src/admin/ladmin.cpp]: `Femal` <= never heard of that gender

File: src/char/char.cpp [hunk 2]
```diff
@@ -265,10 +276,8 @@ AString mmo_char_tostr(struct CharPair *cp)
             p->hair, p->hair_color, p->clothes_color,
             p->weapon, p->shield, p->head_top, p->head_mid, p->head_bottom,
             p->last_point.map_, p->last_point.x, p->last_point.y,
-            p->save_point.map_, p->save_point.x, p->save_point.y, p->partner_id);
-
-    // memos were here (no longer supported)
```
Comment (mekolat) [inline src/char/char.cpp]: @wushin you could've taken karma instead; I would much rather use a variable for that

Comment (wushin) [inline src/char/char.cpp]: @mekolat People still have karma values. I used memos because o11c wiped everyones char memos.

**Comment by wushin** [comment]
Manaplus changes needed to make it work for testing. https://gist.github.com/wushin/39316698a384d4f0157e

**Comment by wushin** [comment]
@charchangesex broken.

**Comment by wushin** [comment]
@mekolat We should also rebase first and squash some of this.

**Comment by mekolat** [comment]
@wushin does `We` include me?

**Comment by wushin** [comment]
@mekolat Nah, I got it if you don't want to mess with it.

**Pull request merged**